src/editing_tools/RedEyeTool.vala

```diff
@@ -70,8 +70,11 @@ public class EditingTools.RedeyeTool : EditingTool {
                 RedeyeInstance.MIN_RADIUS, RedeyeInstance.MAX_RADIUS, 1.0);
 
         public RedeyeToolWindow (Gtk.Window container) {
-            base (container);
+            Object (transient_for: container);
 
```

Comment:
Can you remove this extra new line while we're here?

Reply:
Oh I did not notice. I will. Thanks